Makefile

```diff
@@ -45,6 +45,11 @@ else
   IMAGE_BUILD_CMD = $(ENGINE) build
 endif
 
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+ENVTEST = go run ${PROJECT_DIR}/vendor/sigs.k8s.io/controller-runtime/tools/setup-envtest
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.25
```

Comment:
I'd put this at the top so it's easier to find and harder to forget bumping.

Reply:
With @JoelSpeed we were discussing adding an extra test for checking evntest version and its accordance with another k8s.io dependencies. Would prefer to solve it this way.

Reply:
👍